roles/pgbench/templates/workload.yml.j2

```diff
@@ -14,27 +14,30 @@ spec:
       containers:
       - name: benchmark
         image: "quay.io/cloud-bulldozer/pgbench:latest"
+        env:
+          - name: PGPASSWORD
+            value: '{{ item.1.password }}'
+          - name: uuid
+            value: '{{ uuid }}'
+          - name: pgbench_auth
+            value: >-
+                   -h {{ item.1.host }}
+                   -p {% if item.1.port is defined and item.1.port|int > 0 %} {{ item.1.port }} {% else %} {{ db_port }} {% endif %}
+                   -U {{ item.1.user }}
+          - name: test_user
+            value: '{{ test_user }}'
+{% if es_server %}
```

Comment:
shouldn't this be `es_server|length > 0` ? Because we're by default initializing the var to an empty string?

Reply:
No; this is valid jinja2 logic that will pass if the var is non-empty.
https://jinja.palletsprojects.com/en/2.10.x/templates/#if